.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=$PYTHONPATH:.
```

Comment:
I had to check in the .env file to include this. We are not storing anything confidential so I think it's fine for now